invenio_pidstore/upgrades/pidstore_2014_08_22_initial.py

```diff
@@ -1,93 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
we can't simply remove the upgrades without creating upgrade recipe that removes them first (check invenio-upgrader for an example).